lib/libc/fixedmath/Make.defs

```diff
@@ -1,6 +1,6 @@
 ###########################################################################
 #
-# Copyright 2016 Samsung Electronics All Rights Reserved.
+# Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
Please revert license year to 2016, because it is not the last modified year, but the first written year.